flexbox/src/main/java/com/google/android/flexbox/FlexLine.java

```diff
@@ -95,6 +95,14 @@
     int mLastIndex;
 
     /**
+     * Indicates whether this flex line instance is in a dirty state (needs to be computed before
+     * rendering). This flag is set to true for example an item is added at the position not
+     * visible and the position is prior to the position than the first visible position.
```

Comment:
Yes, that's right.

But after the investigation, it turned out that dirty flag is going to be set to false before scrolling to that position. Because in the normal layout pass in the onLayoutChildren method, the flex lines after the visible potion are going to be cleared (https://github.com/google/flexbox-layout/blob/dev_recyclerview/flexbox/src/main/java/com/google/android/flexbox/FlexboxLayoutManager.java#L773).

So the dirty flag needs to be considered only when scrolling toward start, that means new flex lines will be computed anyway when scrolling toward end.